action.yml

```diff
@@ -24,14 +30,28 @@ inputs:
   required-roles:
     description: "Any user roles which the application expects to exist before migrating. Separate with whitespace if multiple are required, e.g. `admin develop`."
     required: false
+  data-dictionary-repo:
+    description: "Name of the data dictionary repo"
+    required: false
+    default: "data-dictionary"
+  data-dictionary-ref:
+    description: "Ref of the data dictionary repo"
+    required: false
+    default: "master"
+  data-dictionary-workflow:
+    description: "Name of the data dictionary build workflow"
+    required: false
+    default: "build.yml"
 
 runs:
   using: "composite"
   steps:
-    - run: pip3 install -r ${{ github.action_path }}/requirements.txt
+    - run: python -m pip install -r ${{ github.action_path }}/requirements.txt
```

Comment:
In this case it's not necessary as we're not installing Python ourselves, both are internal to Actions runners